index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <link
+    href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,300"
+    rel="stylesheet"
+    type="text/css"
+  />
+  <link rel="stylesheet" href="style.css" />
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Job Board App</title>
+  </head>
+  <body>
+    <ul>
+      <li><a href="default.asp">Home</a></li>
+      <li><a href="news.asp">Blah</a></li>
+      <li><a href="contact.asp">Blah</a></li>
+      <li><a href="about.asp">Blah</a></li>
```

Comment:
When you click on these anchors (<a>) the page attempts to go to these .asp files - which don't exist 😱 It would be great to either add placeholder html files or remove the href attribute for now :)

Reply:
added placekitten links in the mean time, until we have js attributes in place